WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -419,5 +419,33 @@ private async Task<EstimateSmartFeeResponse> GetEstimateSmartFeeAsync(int target
 
 			return feeResponse;
 		}
+
+		[HttpGet("status")]
+		[ProducesResponseType(typeof(StatusResponse), 200)]
+		[ResponseCache(Duration = 10, Location = ResponseCacheLocation.Client)]
```

Comment:
This is to let the browsers know that they can keep it in their local cache for 10 seconds. This does not prevent someone to make requests.

Reply:
Right, added caching! Now the Integration test failing. Do you have an idea of how to reset the cache on demand when testing?

Reply:
Make the cache key as follows:

```c#
var lastFilter = Global.IndexBuilderService.GetLastFilter()`
var cacheKey = $"{nameof(GetStatusAsync)}:{lasFilter.Header.BlockHash}";
```

In this way the cache will be invalidated when a new filter is available.

Reply:
I added. Now if there is a new filter it will invalidate the cache. But still the test fails, see conversation here: https://github.com/zkSNACKs/WalletWasabi/pull/3051#discussion_r376068860

Reply:
@lontivero I removed the filter hash from the cache key. I am planning to add more statuses to this request. A filter-based cache key would cause unnecessary complexity. We have time UptimeRobot querying the status every 1-5 minutes - so do not need to refresh the status immediately when a filter was created.